crates/typst-library/src/loading/cbor.rs

```diff
@@ -8,15 +8,44 @@ use crate::loading::{DataSource, Load};
 
 /// Reads structured data from a CBOR file.
 ///
-/// The file must contain a valid CBOR serialization. Mappings will be
-/// converted into Typst dictionaries, and sequences will be converted into
-/// Typst arrays. Strings and booleans will be converted into the Typst
-/// equivalents, null-values (`null`, `~` or empty ``) will be converted into
-/// `{none}`, and numbers will be converted to floats or integers depending on
-/// whether they are whole numbers.
-///
-/// Be aware that integers larger than 2<sup>63</sup>-1 will be converted to
-/// floating point numbers, which may result in an approximative value.
+/// The file must contain a valid CBOR serialization. The CBOR values will be
+/// converted into corresponding Typst values listed in the
+/// [table below](#conversion).
+///
+/// The function returns a dictionary, an array or, depending on the CBOR file,
+/// another CBOR data type.
+///
+/// # Conversion details { #conversion }
+/// Typst uses [ciborium](https://docs.rs/ciborium/latest/ciborium/#compatibility-with-other-implementations)
+/// to process CBOR data. The ciborium project is wire-compatible with other
+/// implementations in decoding, but not necessarily encoding.
```

Comment:
I see that this is written in the docs, but that seems so odd, and I'm not sure I understand the example on docs.rs with the numbers. If serde_cbor can't decode ciborium's output isn't that necessarily a bug in either one of the two?